test/aws/rails/mailer_test.rb

```diff
@@ -7,13 +7,9 @@ class TestMailer < ActionMailer::Base
   layout nil
 
   def deliverable(options = {})
-    mail(
-      body: options[:body],
-      delivery_method: :ses,
-      from: options[:from],
-      subject: options[:subject],
-      to: options[:to]
-    )
+    mail(delivery_method: :ses, body: '').tap do |m|
+      options.each {|k, v| m.send(k, v) }
+    end
```

Comment:
Making this generic is fine. Could we pass those directly to mail's parameter instead of using send on accessor methods?

```
mail(**{delivery_method: :ses, body: ''}.merge(options))
```

Reply:
I tried that, but that only works for `body` and attributes that are mapped to header fields, but not for `smtp_envelope_from`/`_to`. But now I have made a separate test for SMTP envelope addresses, so this is limitation is no longer a problem.